docs/en/ufs/S3.md

```diff
@@ -279,44 +249,92 @@ alluxio.underfs.s3.upload.threads.max=80
 alluxio.underfs.object.store.service.threads=80
 ```
 
-## S3 Access Control
+## Identity and Access Control of S3 Objects
+
+[S3 identity and access management](https://docs.aws.amazon.com/AmazonS3/latest/dev/s3-access-control.html)
+is very different from the security model of a UNIX-like file system
+(e.g., S3 ACL has no groups and no directory-level setting).
+Alluxio makes a best effort to inherit file owner and permission bits from S3 ACL information.
+Alluxio master loads the ACL information of objects and buckets on the first time when their
```

Comment:
`The Alluxio master loads the ACL information of object and bucket upon initially requesting for their metadata.`
unless you have more specific details about the translation, i dont think the second part of the sentence is necessary since you already said that the master loads the info in the first half and stated that alluxio makes a best effort to inherit permissions in the previous sentence

Reply:
agree , second part is unnecessary details in context here